src/models/companyNumber.model.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const Constants = require('../constants')
+const DynamicsDalService = require('../services/dynamicsDal.service')
+const BaseModel = require('./base.model')
+const LoggingService = require('../services/logging.service')
+const ApplicationLine = require('./applicationLine.model')
+
+module.exports = class Site extends BaseModel {
```

Comment:
This should be called CompanyNumber rather than Site

Reply:
As discussed, since this is currently unused this should be renamed to CompanyDetails and moved into the models/taskList folder with stub methods provided for updateCompleteness() and _isComplete() 

i.e. as per siteNameAndLocation.model.js